packages/client/test/rpc/eth/getFeeHistory.spec.ts

```diff
@@ -0,0 +1,161 @@
+import { hexToBytes } from 'ethereum-cryptography/utils'
+import * as tape from 'tape'
+
+import { baseRequest, gethGenesisStartLondon, params, setupChain } from '../helpers'
+
+import pow = require('./../../testdata/geth-genesis/pow.json')
+
+import type { Chain } from '../../../src/blockchain'
+import type { VMExecution } from '../../../src/execution'
+
+const method = 'eth_feeHistory'
+
+const produceFakeGasUsedBlock = async (execution: VMExecution, chain: Chain, gasUsed: bigint) => {
+  const { vm } = execution
+  const parentBlock = await chain.getCanonicalHeadBlock()
+  const vmCopy = await vm.copy()
+  // Set block's gas used to max
+  const blockBuilder = await vmCopy.buildBlock({
+    parentBlock,
+    headerData: {
+      timestamp: parentBlock.header.timestamp + BigInt(1),
+      gasUsed: bigIntToHex(gasUsed),
+    },
+    blockOpts: {
+      calcDifficultyFromHeader: parentBlock.header,
+      putBlockIntoBlockchain: false,
+    },
+  })
+  blockBuilder.gasUsed = gasUsed
+
+  const block = await blockBuilder.build()
+  await chain.putBlocks([block], false)
+  await execution.run()
+}
+
+tape(`${method}: should return 12.5% increased baseFee if parent block is full`, async (t) => {
+  const { chain, server, execution } = await setupChain(gethGenesisStartLondon(pow), 'powLondon')
+  const gasUsed = bytesToBigInt(hexToBytes(pow.gasLimit))
+
+  await produceFakeGasUsedBlock(execution, chain, gasUsed)
+  const req = params(method, ['0x2', 'latest', []])
+
+  const expectRes = (res: any) => {
+    const [, previousBaseFee, nextBaseFee] = res.body.result.baseFeePerGas as [
+      string,
+      string,
+      string
+    ]
+    const increase =
+      Number(
+        (1000n *
+          (bytesToBigInt(hexToBytes(nextBaseFee)) - bytesToBigInt(hexToBytes(previousBaseFee)))) /
+          bytesToBigInt(hexToBytes(previousBaseFee))
+      ) / 1000
+
+    t.equal(increase, 0.125)
+  }
+
+  await baseRequest(t, server, req, 200, expectRes)
+})
+
+tape(`${method}: should return 12.5% decreased base fee if the block is empty`, async (t) => {
+  const { chain, server, execution } = await setupChain(gethGenesisStartLondon(pow), 'powLondon')
+  const gasUsed = BigInt(0)
+
+  await produceFakeGasUsedBlock(execution, chain, gasUsed)
+  const req = params(method, ['0x2', 'latest', []])
+
+  const expectRes = (res: any) => {
+    const [, previousBaseFee, nextBaseFee] = res.body.result.baseFeePerGas as [
+      string,
+      string,
+      string
+    ]
+    const increase =
+      Number(
+        (1000n *
+          (bytesToBigInt(hexToBytes(nextBaseFee)) - bytesToBigInt(hexToBytes(previousBaseFee)))) /
+          bytesToBigInt(hexToBytes(previousBaseFee))
+      ) / 1000
+
+    t.equal(increase, -0.125)
+  }
+
+  await baseRequest(t, server, req, 200, expectRes)
+})
+
+tape(
+  `${method}: should return initial base fee if the block number is london hard fork`,
+  async (t) => {
+    const initialBaseFee = 1000000000n
```

Comment:
This const should probably be retrieved from common

Reply:
This can be retrieved with `common.param('gasConfig', 'initialBaseFee')` (so it's there, guess you have overlooked).